src/components/labels/Labels.tsx

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState} from 'react';
+import {
+  getLabels,
+  LabelsResponse,
+  Label as ImageLabel,
+} from 'src/resources/TagResource';
+import {Label} from '@patternfly/react-core';
+
+export default function Labels(props: LabelsProps) {
+  const [labels, setLabels] = useState<ImageLabel[]>([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const labelsResp: LabelsResponse = await getLabels(
+          props.org,
+          props.repo,
+          props.digest,
+        );
+        setLabels(labelsResp.labels);
+      } catch (error: any) {
+        console.log('Unable to get labels: ', error);
```

Comment:
use `console.error`

Reply:
This is going to be removed entirely in a future PR anyway. Unless you're thinking of keeping this in when error handling gets implemented?

Reply:
it would be good to keep `console.error` for dev purposes. When we do a `npm build` there is a webpack plugin that Dave had which will remove all `console.*` lines. Another thing we can do is route all `console.error` logs to something like sentry where we can see what errors are happening "in the wild" with users which we can't replicate

Reply:
sounds good, changed to console.error